src/tribler/core/components/torrent_checker/torrent_checker/dataclasses.py

```diff
@@ -1,14 +1,17 @@
 import time
 from dataclasses import dataclass, field
-from typing import List
+from typing import List, Tuple
 
 import human_readable
 
 from tribler.core.utilities.unicode import hexlify
 
 
-TOLERABLE_TIME_DRIFT = 60  # one minute
-HOUR = 60 * 60
+MINUTE = 60
+HOUR = MINUTE * 60
+TOLERABLE_TIME_DRIFT = MINUTE  # When receiving health from another peer, how far the timestamp can be in the future?
+TORRENT_CHECK_WINDOW = MINUTE  # When asking multiple trackers in parallel, we ignore this time difference in responses
+HEALTH_FRESHNESS_SECONDS = 4 * HOUR  # Number of seconds before a torrent health is considered stale. Default: 4 hours
```

Comment:
This is great that you have added this comments 👍

Reply:
I added a few methods to HealthInfo class that should make the code easier to read: `health.just_checked()`, `health.old()`, `health.older_than(other)`, `health.much_older_than(other)`